src/tempo.py

```diff
@@ -163,3 +158,36 @@ def is_ready(self):
         except (CalledProcessError, IndexError):
             return False
         return out == "ready"
+
+    @staticmethod
+    def _build_receivers_config(receivers: Sequence[ReceiverProtocol]):  # noqa: C901
+        receivers_set = set(receivers)  # convert to set for faster lookup
+
+        config = {}
+
+        if "zipkin" in receivers_set:
+            config["zipkin"] = None
+        if "opencensus" in receivers_set:
+            config["opencensus"] = None
+
+        otlp_config = {}
+        if "otlp_http" in receivers_set:
+            otlp_config["http"] = None
```

Comment:
This might still need to be filled? Unless I miss something which is possible at this point ;)

Reply:
mmm, what do you mean? as far as I can tell from https://github.com/open-telemetry/opentelemetry-collector/tree/overlord/receiver , the minimal config is `receiver-name: null`.

Reply:
Might've been under the impression that this is the place where we enter port numbers into the config. If that's just the structure, my comment can be ignored.

Reply:
might be a difference between tempo v1 and v2. Couldn't find a clear answer in the docs.
For now it seems to be working with cos-lite so we'll fix and validate this in the next PR